src/govuk/settings/_typography-font.scss

```
@@ -43,13 +56,23 @@ $govuk-font-family: if(
 ///
 /// @type List
 /// @access public
+/// @deprecated Will be removed in v5.0 with the rest of the compatibility mode
+/// suite of tools and settings
 
 $govuk-font-family-tabular: if(
   $govuk-use-legacy-font,
   $govuk-font-family-nta-tabular,
   false
 ) !default;
 
+// Only show the deprecation warning if user is setting $govuk-font-family-tabular
+// manually instead of automatically via $govuk-use-legacy-font
+@if $govuk-font-family-tabular != false and $govuk-use-legacy-font == false {
+  @warn "$govuk-font-family-tabular is deprecated. " +
+    "From version 5.0, Govuk-frontend will not support this method of " +
+    "specifying tabular data";
```

This isn't really anything to do with data, suggest something like:

```suggestion
    "From version 5.0, GOV.UK Frontend will not support using a separate " +
    "font-face for tabular numbers";
```